.github/workflows/dev_module_build.yml

```diff
@@ -51,6 +55,8 @@ jobs:
     name: Set MODULES_MODULE_TAG
     outputs:
       modules_module_tag: ${{ steps.modules_module_tag.outputs.MODULES_MODULE_TAG }}
+      edition: ${{ steps.modules_module_tag.outputs.MODULE_EDITION }}
```

Comment:
Seems a typo, `dev_setup_build` uses ouputs.module_edition:

```
      MODULE_EDITION: ${{needs.set_vars.outputs.module_edition}}
```

Reply:
Yes, it should be module_edition. Fixed